packages/modules/src/modules/banners/charityAppeal/CharityAppealBanner.tsx

```diff
@@ -210,22 +209,24 @@ export const getCharityAppealBanner = (
     reminderTracking,
     onCloseClick,
     content,
-    email,
     fetchEmail,
 }: BannerRenderProps) => {
     const [isReminderOpen, setIsReminderOpen] = useState(false);
     const [fetchedEmail, setFetchedEmail] = useState<string | undefined>(undefined);
-    const fetchEmailDefined = defineFetchEmail(email, fetchEmail);
 
     const onReminderCtaClick = () => {
         reminderTracking.onReminderCtaClick();
 
-        fetchEmailDefined().then(resolvedEmail => {
-            if (resolvedEmail) {
-                setFetchedEmail(resolvedEmail);
-            }
+        if (fetchEmail) {
```

Comment:
I'm slightly unsure of what the user will see here if `fetchEmail` isn't defined?

Reply:
Good question. If `fetchEmail` isn't defined then we display a text input and a cta.
If it is defined, and it returns an email, we only display a cta.
Either way, `setIsReminderOpen` is always called, causing the reminder signup box to be rendered.

Reply:
From storybook, here are the two versions in the epic (the banners follow a similar design):
<img width="627" alt="Screenshot 2023-04-20 at 08 18 13" src="https://user-images.githubusercontent.com/1513454/233290088-61a9993c-4102-4aad-a104-879d4bf4ae68.png">
<img width="627" alt="Screenshot 2023-04-20 at 08 18 17" src="https://user-images.githubusercontent.com/1513454/233290095-e180a185-8cf7-4e01-9fee-b9f12dbbdd99.png">

Reply:
Ah great, thanks for explaining 👍